phpcs.xml.dist

```diff
@@ -0,0 +1,45 @@
+<?xml version="1.0"?>
+<ruleset name="Beans">
+    <description>The code standard for Beans.</description>
+    <file>.</file>
+    <exclude-pattern>dist/*</exclude-pattern>
+    <exclude-pattern>node_modules/*</exclude-pattern>
+    <exclude-pattern>vendor/*</exclude-pattern>
+    <arg name="extensions" value="php"/>
+    <arg value="sp"/>
+    <arg name="report" value="full"/>
+    <arg name="report" value="summary"/>
+    <arg name="report" value="source"/>
```

Comment:
Summary and source reports help to summarise the state of the project, beyond just the list of issues per file.